application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -398,7 +404,13 @@
 .info
 {
   float: left;
-}</code>
+}
+
+#xpollSaveForm .noBorder {
+  border-right: solid 1px #FFF;
+  border-left: solid 1px #FFF;
+  padding-top: 30px;
+ }</code>
```

Comment:
It looks like we have a wrong indentation here. There shouldn't be an empty space before `}`.